packages/common/src/intl/locales/en/distribution.json

```diff
@@ -32,5 +32,8 @@
   "message.deleted-requisitions": "Deleted {{number}} requisitions",
   "message.select-rows-to-delete": "Select rows to delete",
   "messages.click-to-return-to-requisitions": "Unable to find a requisition with that ID. Click OK to return to the requisition list",
-  "message.deleted-lines": "Deleted {{number}} lines"
+  "message.deleted-lines": "Deleted {{number}} lines",
+  "message.confirm-status-as": "Confirm status as $t({{status}})?",
+  "message.shipment-saved": "Shipment saved 🥳",
```

Comment:
With the modal and the loading spinner this could work well as just 'Success' or some such.
As you've already said.. I don't think we need the `Shipment saved` notification so much now.